Model/Quote/GetCartLineItems.php

```diff
@@ -159,4 +273,52 @@ private function addCustomOptions(array $customOption, array $lineItem): array
         $lineItem['line_item_properties'][$label] = $value['value'] ?? '';
         return $lineItem;
     }
+
+    /**
+     * Takes in a bundle product line item and adds the items in the bundle to the line
+     * item as line item properties
+     *
+     * @param CartItemInterface $item
+     * @param array $lineItem
+     * @return array
+     */
+    private function addBundleOptions(CartItemInterface $item, array $lineItem): array
+    {
+        $options = $this->bundleType->getOptionsCollection($item->getProduct());
+        $children = $item->getChildren();
+        $lineItem['line_item_properties'] = [];
+        foreach (array_values($options->getItems()) as $i => $option) {
+            $childItem = $children[$i] ?? null;
+            if (!$childItem) {
```

Comment:
nit: We could avoid using the ternary operator here to simplify the code:
if(is_null($children[$i])) {
continue;
}

$chilItem = $children[$i];

Reply:
?? is a nullish coalescing operator. It is needed here or PHP would through an error if index `$i` is not found.